tests/cluster-check.sh

```diff
@@ -45,42 +49,42 @@ set -e
 if test "$scheduler" = "slurm" ; then
     cat > job1.sh <<EOF
 #!/bin/bash
-srun sleep 960
+srun sleep ${_sleepjob1}
 touch job1.done
 EOF
     cat > job2.sh <<EOF
 #!/bin/bash
-srun sleep 960
+srun sleep ${_sleepjob2}
 touch job2.done
 EOF
 
     chmod +x job1.sh job2.sh
     rm -f job1.done job2.done
 
-    sbatch -N 2 ./job1.sh
-    sbatch -N 2 ./job2.sh
+    sbatch -N 1 ./job1.sh
+    sbatch -N 1 ./job2.sh
 
 elif test "$scheduler" = "sge" ; then
     # get the slots per node count of the first real node (one with a
     # architecture type of lx-?), so that we can reserve an enitre
     # node's worth of slots at a time.
     ppn=` qhost | grep 'lx-' | head -n 1 | sed -n -e 's/[^[:space:]]*[[:space:]]\+[^[:space:]]*[[:space:]]\+\([0-9]\+\).*/\1/p'`
```

Comment:
If there are not yet compute nodes, the `ppn` is an empty value, so `count=$((ppn))` results `0` and the job will fail with: `Unable to run job: "job" rejected: pe range must be greater than zero`.

We could check the `count` value and fail with a more clear message or perform a loop waiting for the compute nodes.

Reply:
Improved the way we retrieve the slots per node for the rare cases (seen in our tests) where the first compute node was not there when trying to submit the jobs.